bundles/net.tourbook/src/net/tourbook/tour/DialogQuickEdit.java

```diff
@@ -81,7 +82,6 @@
    private boolean                  _isWindSpeedManuallyModified   = false;
    private int[]                    _unitValueWindSpeed;
    private float                    _unitValueDistance;
-   private float                    _unitValueTemperature;
 
    /*
     * UI controls
```

Comment:
Can you display the device avg temperature also in the quick edit dialog, when there is only one value, it is irritating

![image](https://user-images.githubusercontent.com/1283445/157815667-be94bf16-c4db-499c-999f-818c0083c10f.png)

Reply:
Sure, I will add that in my next PR

Reply:
Fixed in my [PR](https://github.com/mytourbook/mytourbook/pull/485)

![image](https://user-images.githubusercontent.com/25235665/158253032-35568e2d-abe0-48eb-8d3f-25e6f777e95b.png)